pyannote/audio/labeling/tasks/__init__.py

```diff
@@ -34,3 +34,6 @@
 from .speaker_change_detection import SpeakerChangeDetection
 from .domain_classification import DomainClassification
 from .resegmentation import Resegmentation
+from .multilabel import Multilabel
+
+from .multilabel import derives_label
```

Comment:
`derives_label` does not need to be imported here, right?

Reply:
We need it, so that we can use it in pyannote/audio/applications/multilabel.py

During the validation step, we also need to "compute" the meta-classes. I imported it to avoid code duplication.

Reply:
Can we make it a static method of the Multilabel class instead?

Reply:
Sure

Reply:
Solved